events.go

```diff
@@ -177,3 +185,15 @@ func (eo *eventObserver) Subscribe(ids []int64, handler func(*Event)) Unsubscrib
 	}
 
 }
+
+func (eo *eventObserver) SubscribeAll(handler func(*Event)) Unsubscribe {
+	eo.lock.Lock()
+	defer eo.lock.Unlock()
+
+	eo.listener = handler
```

Comment:
Multiple subscribe calls will override each other.

Reply:
Okay, yes, so this comment helped me finally make sense of your other code :P 

I'm struggling to name this guy. Should we call them `globalListeners`?